docs/helm-charts.md

```diff
@@ -1,17 +1,21 @@
 # Helm Charts
 
-k0s supports deploying applications with Helm charts in two ways:
+Helm charts is one of two methods you can use to run k0s with your preferred extentions (the other being [Manifest Deployer](manifests.md), which is included with k0s).
```

Comment:
"Helm charts is one of two methods" sounds wrong. 

Anyway, both of the ways refer to helm charts. The first method is by using the `helm` tool, the second method is by using k0s.yaml and have k0s do the applying of the helm charts.

Also:

```suggestion
Helm charts is one of two methods you can use to run k0s with your preferred extensions (the other being [Manifest Deployer](manifests.md), which is included with k0s).
```

Reply:
Killed the filed lead-in.